server/src/main/java/org/apache/iotdb/db/qp/physical/crud/LastQueryPlan.java

```diff
@@ -80,16 +77,12 @@ public void setExpression(IExpression expression) throws QueryProcessException {
     if (isValidExpression(expression)) {
       super.setExpression(expression);
     } else {
-      throw new QueryProcessException("Only '>' and '>=' are supported in LAST query");
+      throw new QueryProcessException("Only time filters are supported in LAST query");
     }
   }
 
   // Only > and >= are supported in time filter
   private boolean isValidExpression(IExpression expression) {
-    if (expression instanceof GlobalTimeExpression) {
-      Filter filter = ((GlobalTimeExpression) expression).getFilter();
-      return filter instanceof TimeGtEq || filter instanceof TimeGt;
-    }
-    return false;
+    return expression instanceof GlobalTimeExpression;
```

Comment:
Only time filters are supported in LAST query.
What about other time filter like `TimeEq`, `TimeNotEq` or `TimeIn` ?

Reply:
Good question.

Time filter is used in 2 scenarios,
1. judge if the last value cache satisfies the filter
2. being passed to the LastPointReader

Any time filter works with the two scenarios above.

Reply:
Maybe you could add `TimeEq`, `TimeNotEq` and `TimeIn` case in IT.